documentation/modules/upgrading-mtv-ui.adoc

```diff
@@ -14,13 +14,58 @@ You must upgrade to the next release without skipping a release, for example, fr
 
 .Procedure
 
-. Upgrade the {operator-name}.
+. Change the update channel:
 +
 ifeval::["{build}" == "upstream"]
-See link:https://docs.okd.io/latest/operators/admin/olm-upgrading-operators.html[Upgrading installed Operators] in the {ocp} documentation.
+See link:https://docs.okd.io/latest/operators/admin/olm-upgrading-operators.html#olm-changing-update-channel_olm-upgrading-operators[Changing update channel] in the {ocp} documentation.
 endif::[]
 ifeval::["{build}" == "downstream"]
-See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html-single/operators/index#olm-upgrading-operators[Upgrading installed Operators] in the {ocp} documentation.
+See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html-single/operators/index#olm-changing-update-channel_olm-upgrading-operators[Changing update channel] in the {ocp} documentation.
+endif::[]
+
+. Restart the `CatalogSource` pod:
+
+.. In the the {ocp-short} web console, navigate to the *Operators* -> *Installed Operators* -> *Operator details* page. On the *Subscriptions* tab, note the name in the *CatalogSource* field.
+..  From the command line, run the following command:
++
+[source,terminal]
+----
+$ oc get pod -n openshift-marketplace
```

Comment:
If I am not mistaken the command runs on OKD cluster is 'kubectl' and not 'oc'
@apinnick could you please confirm?

Reply:
@sbeskin-redhat  Please replace "oc" with the oc attribute so that it renders as kubectl.

Reply:
@apinnick  The user only has the catalog source name from the UI. The name of the pod is different.